CHANGELOG.md

```diff
@@ -2,110 +2,144 @@
 
 ## 4.0.0 - UNRELEASED
 
-- BREAKING: Map `multi-merge` utility and all `add-*` functions
-  for adding one or more token maps (eg `add-sizes`, `add-colors`, etc)
-  now perform a deep merge of the token maps.
-  Previously a shallow merge was performed,
-  [which could result in removing nested values from maps](https://github.com/oddbird/accoutrement/issues/50).
+This is a major update, moving over to Sass modules,
+and removing some features that no longer seem necessary.
 
-- [Tokens][token]:
+In addition to fixing a bug with the handling of
+non-alias `#` symbols, and implementing deep map merges
+where appropriate, we also added some functionality around
+compiling maps for export, and working with different
+color palettes.
 
-  - NEW: `map-compile` and `map-compile-with` functions for compiling
-    tokens in Accoutrement maps and formatting them for static exports.
+- BREAKING: Requires Sass modules, and other recent features of Dart Sass.
+  Legacy versions of Sass (Node & Ruby) are no longer supported.
+- BREAKING: The old "init" module, which provided lightweight
+  browser normalization, is no longer included. We now recommend
+  using [CSS Remedy](https://github.com/jensimmons/cssremedy). (_Note: If
+  making the switch from Accoutrement/Init to CSS Remedy, you may notice changes
+  in vertical spacing and the displaying of SVGs._)
```

Comment:
This is very helpful. Is there anything we could link to -- or hints to provide -- for people making that migration?

Reply:
@jgerigmeyer Made this a bit [more specific](https://github.com/oddbird/accoutrement/pull/97/commits/bbff01555d64914dd49dc684dec1edd618a16567). Does this work better?